wordpress/wp-content/plugins/cds-base/classes/Modules/Cleanup/Menus.php

```diff
@@ -25,30 +25,45 @@ public function removeMenuPages(): void
             return;
         }
 
-        global $menu, $submenu;
+        global $menu;
 
         /* add items to keep here */
         $allowed = [
-            __('Pages'),
-            __('Posts'),
-            __('Articles', 'cds-snc'),
-            __('Bulk', 'cds-snc'),
-            __('Users'),
-            __('Settings'),
-            __('Media')
+            // English menu items
+            "Dashboard",
+            "Articles",
+            "Media",
+            "Pages", // same in FR
+            "Menus", // same in FR
+            "Users",
+            "Settings",
+            "Bulk Send",
+
+            // French menu items
+            "Tableau de bord",
+            "Les Articles",
+            "Média",
+            "Utilisateurs",
+            "Réglages",
+            "Envoyer en masse",
```

Comment:
Note that I removed the i18n wrapper here because we aren't using these strings in the interface, but to do a logical comparison.